src/i_winmusic.c

```diff
@@ -76,6 +76,15 @@ static byte channel_volume[MIDI_CHANNELS_PER_TRACK];
 static float volume_factor = 0.0f;
 static boolean update_volume = false;
 
+typedef enum
+{
+    STATE_PLAYING,
+    STATE_STOPPED,
```

Comment:
I think it would be more logical to have `STATE_STOPPED` first, as kind of "default".

Reply:
https://github.com/chocolate-doom/chocolate-doom/pull/1603/commits/6dcdf5d96eb66a2c3b23a829598ed8f6a59d8c85